cmd/soroban-cli/src/commands/tx/edit.rs

```diff
@@ -0,0 +1,186 @@
+use std::{
+    env,
+    fs::{self},
+    io::{stdin, Cursor, IsTerminal, Write},
+    path::PathBuf,
+    process::{self},
+};
+
+use serde_json::json;
+use stellar_xdr::curr;
+
+use crate::{commands::global, print::Print};
+
+const DEFAULT_JSON: &str = r#"{
```

Comment:
I'm not sure why we want this behavior of 
`stellar tx edit` (I don't think anybody would edit transactions manually)
Why doesn't it instead wait for transaction to be pasted in the terminal like other commands?

Reply:
Again, I don't think keeping waiting on stdin is the best user experience. I think we should either error or give the user a default. Given `tx edit` will have JSON schema autocompletion, it seems like a good default.

@janewang wdyt? This case is where users just type `stellar tx edit` without stdin, so we're currently opening a default JSON structure that allows users to go from there.